shell/platform/embedder/embedder.h

```diff
@@ -336,9 +336,18 @@ typedef struct {
   size_t width;
   /// Physical height of the window.
   size_t height;
+} FlutterWindowMetricsEvent;
```

Comment:
This is a breaking change, which we don't generally allow in embedder.h. If we're changing the structs, we need new structs and new APIs using them, and the old path needs to do something reasonable (presumably translate under the hood to the new structs then call through).